shpkpr/commands/cmd_bluegreen.py

```diff
@@ -6,20 +6,34 @@
 import time
 
 # local imports
+from shpkpr import exceptions
 from shpkpr.cli import arguments
 from shpkpr.cli import options
 from shpkpr.cli.entrypoint import CONTEXT_SETTINGS
 from shpkpr.cli.logger import pass_logger
 from shpkpr.template import load_values_from_environment
 from shpkpr.template import render_json_template
+from shpkpr.marathon import DeploymentFailed
+from shpkpr.marathon_lb import SwapApplicationTimeout
 from shpkpr.marathon_lb import fetch_previous_deploys
 from shpkpr.marathon_lb import prepare_deploy
-from shpkpr.marathon_lb import safe_resume_deploy
 from shpkpr.marathon_lb import select_last_deploy
 from shpkpr.marathon_lb import swap_bluegreen_apps
 from shpkpr.marathon_lb import validate_app
 
 
+class DualStackAlreadyExists(exceptions.ShpkprException):
+    """Raised if a deploy is attempt whilst both a blue and green stack
+    already exist in Marathon
+    """
+    exit_code = 2
+
```

Comment:
So this is actually a click convention, and doing so here fits in with the rest of the application.

Explained here http://click.pocoo.org/6/exceptions/